run_meson_command_tests.py

```diff
@@ -141,11 +146,17 @@ def test_meson_installed(self):
         # distutils complains that prefix isn't contained in PYTHONPATH
         os.environ['PYTHONPATH'] = os.path.join(str(pylibdir), '')
         os.environ['PATH'] = str(bindir) + os.pathsep + os.environ['PATH']
-        self._run(python_command + ['setup.py', 'install', '--prefix', str(prefix)])
-        # Fix importlib-metadata by appending all dirs in pylibdir
-        PYTHONPATHS = [pylibdir] + [x for x in pylibdir.iterdir()]
-        PYTHONPATHS = [os.path.join(str(x), '') for x in PYTHONPATHS]
-        os.environ['PYTHONPATH'] = os.pathsep.join(PYTHONPATHS)
+        if (has_python_module('pip')):
+            self._run(python_command + ['-m', 'pip', 'install', '--prefix', str(prefix), '.'])
+        elif (has_python_module('gpep517')):
+            self._run(python_command + ['-m', 'gpep517', 'install-from-source', '--destdir', str(prefix)])
```

Comment:
destdir != prefix

It is equivalent to pip install --root.

There is already a --prefix option for it too...